scripts/main.sh

```diff
@@ -6,10 +6,6 @@ set -o pipefail
 
 script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
-if [[ -n $NPM_TOKEN ]]; then
-  npm config set //registry.npmjs.org/:_authToken "${NPM_TOKEN}"
```

Comment:
Yarn was able to read the token directly from the environment variable; no config setting step was needed.